rpc/internal/service.go

```diff
@@ -0,0 +1,204 @@
+package internal
+
+import (
+	"context"
+
+	"github.com/evergreen-ci/poplar"
+	"github.com/golang/protobuf/ptypes"
+	"github.com/pkg/errors"
+	"google.golang.org/grpc"
+)
+
+func AttachService(registry *poplar.RecorderRegistry, s *grpc.Server) error {
+	srv := &poplarService{
+		registry: registry,
+	}
+
+	RegisterPoplarMetricsRecorderServer(s, srv)
+
+	return nil
+}
+
+type poplarService struct {
+	registry *poplar.RecorderRegistry
+}
+
+func (s *poplarService) CreateRecorder(ctx context.Context, info *CreateOptions) (*EventResponse, error) {
+	_, err := s.registry.Create(info.Name, info.Export())
+	if err != nil {
+		return nil, errors.WithStack(err)
+	}
+
+	return &EventResponse{Name: info.Name, Status: true}, nil
```

Comment:
It looks like the `Status` field is uneccessary, it is always set to true. If there is an error, nil is returned since the return type is a pointer to `EventResponse`.

Reply:
yeah, but it's more awkward to not have it, and it will default to false in other constructions.

Reply:
ack